src-built-in/components/myApps/src/stores/StoreActions.js

```diff
@@ -1,5 +1,9 @@
-import { getStore } from './LauncherStore'
-
+import {getStore} from "./LauncherStore";
+import AppDirectory from "../modules/AppDirectory";
```

Comment:
same comments as the other file that looks exactly like this. The data thing is just weird.